.drone.star

```diff
@@ -79,6 +82,65 @@ def unitTestPipeline(ctx):
         },
     }]
 
+def pnpmlint(ctx):
+    pipelines = []
+
+    result = {
+        "kind": "pipeline",
+        "type": "docker",
+        "name": "lint",
+        "steps": skipIfUnchanged(ctx, "lint") +
+                 installPnpm() +
+                 lint(),
+        "trigger": {
+            "ref": [
+                "refs/heads/master",
+                "refs/heads/stable-*",
+                "refs/tags/**",
+                "refs/pull/**",
+            ],
+        },
+    }
+
+    pipelines.append(result)
+
+    return pipelines
+
+def skipIfUnchanged(ctx, type):
```

Comment:
I would recommend to implement this in next iterations.

Reply:
If so I will remove it